control/Director.php

```diff
@@ -351,12 +351,14 @@ protected static function handleRequest(SS_HTTPRequest $request, Session $sessio
 		if(isset($_REQUEST['debug'])) Debug::show($rules);
 
 		foreach($rules as $pattern => $controllerOptions) {
-			if(is_string($controllerOptions)) {
+			if(is_string($controllerOptions) && $controllerOptions) {
```

Comment:
Can you explain why this was necessary? Its changing the core routing, so want to ensure its solid